test/WebJobs.Script.Tests/Scale/FunctionsScaleManagerTests.cs

```diff
@@ -1,387 +0,0 @@
-            _targetScalerManagerMock = new Mock<ITargetScalerManager>(MockBehavior.Strict);
-            _targetScalerManagerMock.Setup(p => p.GetTargetScalers()).Returns(() => _targetScalers);
-            _concurrencyStatusRepositoryMock = new Mock<IConcurrencyStatusRepository>(MockBehavior.Strict);
-            _concurrencyStatusRepositoryMock.Setup(p => p.ReadAsync(It.IsAny<CancellationToken>())).ReturnsAsync(
-                new HostConcurrencySnapshot()
-                {
-                    FunctionSnapshots = new Dictionary<string, FunctionConcurrencySnapshot>()
-                    {
-                        { "func1", new FunctionConcurrencySnapshot() { Concurrency = 1 } }
-                    }
-                });
-
-            _functionsHostingConfigOptions = Options.Create(new FunctionsHostingConfigOptions());
-
-            _environment = new TestEnvironment();
-
-            _scaleManager = new FunctionsScaleManager(_monitorManagerMock.Object, _metricsRepositoryMock.Object, _targetScalerManagerMock.Object, _concurrencyStatusRepositoryMock.Object,
-                _functionsHostingConfigOptions, _environment, _loggerFactory);
-        }
-
-        [Theory]
-        [InlineData(0, ScaleVote.None)]
-        [InlineData(1, ScaleVote.ScaleIn)]
-        public async Task GetScaleStatus_NoMonitors_ReturnsExpectedStatus(int workerCount, ScaleVote expected)
-        {
-            var context = new ScaleStatusContext
-            {
-                WorkerCount = workerCount
-            };
-            var status = await _scaleManager.GetScaleStatusAsync(context);
-
-            Assert.Equal(expected, status.Vote);
-        }
-
-        [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task GetScaleStatus_ReturnsExpectedResult(bool tbsEnabled)
-        {
-            var context = new ScaleStatusContext
-            {
-                WorkerCount = 3
-            };
-
-            var monitor1 = new TestScaleMonitor<TestScaleMetrics1>("func1-test-test", "func1");
-            monitor1.Status = new ScaleStatus
-            {
-                Vote = ScaleVote.ScaleIn
-            };
-            var monitor2 = new TestScaleMonitor1();
-            monitor2.Status = new ScaleStatus
-            {
-                Vote = ScaleVote.ScaleOut
-            };
-            var monitor3 = new TestScaleMonitor1();
-            monitor3.Status = new ScaleStatus
-            {
-                Vote = ScaleVote.ScaleIn
-            };
-
-            List<IScaleMonitor> monitors = new List<IScaleMonitor>
-            {
-                monitor1
-            };
-            if (!tbsEnabled)
-            {
-                monitors.Add(monitor2);
-                monitors.Add(monitor3);
-            }
-            _monitorManagerMock.Setup(p => p.GetMonitors()).Returns(monitors);
```

Comment:
Is there test coverage for these scenarios in WebJobs SDK?

Reply:
yes: https://github.com/Azure/azure-webjobs-sdk/blob/dev/test/Microsoft.Azure.WebJobs.Host.UnitTests/Scale/ScaleManagerTests.cs